src/shared/analytics/mixpanel.ts

```diff
@@ -33,6 +34,19 @@ class DeviceIdStore extends PersistentStore<string | undefined> {
   }
 }
 
+function omitNullParams<T extends Record<string, unknown>>(
+  params: T
+): Partial<T> {
+  return produce(params, (draft) => {
+    const keys = Object.keys(draft);
+    for (const key of keys) {
+      if (draft[key] == null) {
```

Comment:
what about empty arrays?

Reply:
We discussed only `name: null` cases as bad

Reply:
This is because if Mixpanel's limitations